src/bundles/CaseStudy/redux/modules/selector.js

```diff
@@ -1,53 +1,68 @@
 import { createSelector } from 'reselect';
```

Comment:
Have moved all common redux modules to a common place. Still ways away from fixing the tests though.